examples/style_sheets/plot_solarizedlight2.py

```diff
@@ -6,19 +6,21 @@
 This shows an example of "Solarized_Light" styling, which
 tries to replicate the styles of:
 
- - `<http://ethanschoonover.com/solarized>`__
- - `<https://github.com/jrnold/ggthemes>`__
- - `<http://pygal.org/en/stable/documentation/builtin_styles.html#light-solarized>`__
+- http://ethanschoonover.com/solarized
+- https://github.com/jrnold/ggthemes
+- http://pygal.org/en/stable/documentation/builtin_styles.html#light-solarized
+- http://www.pygal.org/en/stable/documentation/builtin_styles.html#light-solarized
```

Comment:
Seems like a duplicated link here?

Reply:
oops, fixed, the version without www is the broken one.